README.md

```diff
@@ -102,6 +102,9 @@ where
 
 ## Install KubeVirt on an existing cluster
 
+Make sure you have a kubeconfig file that's available for ```kubectl``` to use.
```

Comment:
Issue 100 was about missing kubectl command, I would mention in addition that `kubectl` has to be ensured.